sunpy/spectra/spectrogram.py

```diff
@@ -523,6 +523,7 @@ def plot(self, figure=None, overlays=[], colorbar=True, vmin=None,
         xa.set_major_formatter(
             FuncFormatter(self.time_formatter)
         )
+        log_y_axis = True  # Default is True
```

Comment:
import collections

Const = collections.namedtuple('Const', 'log_y_axis')
const = Const('True')

const.log_y_axis

something like this?